utils/generate-commands-json.py

```diff
@@ -69,8 +69,8 @@ def convert_entry_to_objects_array(cmd, docs):
     # The command's value is ordered so the interesting stuff that we care about
     # is at the start. Optional `None` and empty list values are filtered out.
     value = OrderedDict()
-    value['summary'] = docs.pop('summary')
-    value['since'] = docs.pop('since')
+    set_if_not_none_or_empty(value, 'summary', docs.pop('summary', None))
+    set_if_not_none_or_empty(value, 'since', docs.pop('since', None))
```

Comment:
Perhaps only use None when the group is `module`.

Reply:
Are you suggesting something like below?
```py
    group = docs.pop('group')
    if group == 'module':
        set_if_not_none_or_empty(value, 'summary', docs.pop('summary', None))
        set_if_not_none_or_empty(value, 'since', docs.pop('since', None))
    else:
        value['summary'] = docs.pop('summary')
        value['since'] = docs.pop('since')
    value['group'] = group
```
But it seems to have no difference to
```py
    set_if_not_none_or_empty(value, 'summary', docs.pop('summary', None))
    set_if_not_none_or_empty(value, 'since', docs.pop('since', None))
```

Reply:
For non-module commands, "summary" and "since" are required.